src/cs/Ssh.Tcp/Services/PortForwardingService.cs

```diff
@@ -788,7 +788,8 @@ public PortForwardingService(SshSession session)
 
 		request.FailureReason = portForwardRequest.FailureReason;
 		request.FailureDescription = portForwardRequest.FailureDescription;
-		if (request.FailureReason != SshChannelOpenFailureReason.None || !request.IsRemoteRequest)
+		if (request.FailureReason != SshChannelOpenFailureReason.None ||
+			!request.IsRemoteRequest || !AcceptLocalConnectionsForForwardedPorts)
```

Comment:
The name of the property `AcceptLocalConnectionsForForwardedPorts` doesn't quite work for this side, where normally the connection would be _forwarded to a local port_, rather than _accepted by listening on a local port_. So it might be better to create a separate property.

Reply:
Thanks, will create a new property: ForwardConnectionsToLocalPort